processor/processor.go

```diff
@@ -1176,6 +1219,7 @@ func (proc *Handle) getTransformationMetrics(transformerResponses []transformer.
 			"destName":   commonMetaData.DestinationType,
 			"statusCode": strconv.Itoa(failedEvent.StatusCode),
 			"stage":      pu,
+			"isDropped":  isDropped,
```

Comment:
Why don't we use the actual state instead (aborted, filtered)?

Reply:
there's some confusion between different teams using different vocabulary.
Also, filtered might mean dropped at a point and considered a success at some other point.
But `isDropped` - true/false can only mean one thing

Reply:
Since filtered is not a proc_error, can't we just not include them in this metric? 
If we want to capture a counter for filtered events in the future we could use another measurement name altogether. Adding filtered events in `proc_error_counts` using an additional label seems to be a bit unintuitive to me

Reply:
true
pushed some changes
please take another look